modeltranslation/manager.py

```diff
@@ -447,7 +448,25 @@ def values_list(self, *fields, flat=False, named=False):
         if not fields:
             # Emulate original queryset behaviour: get all fields that are not translation fields
             fields = self._get_original_fields()
-        clone = self._values(*fields, prepare=True, selects_all=selects_all)
+
+        field_names = {f for f in fields if not hasattr(f, 'resolve_expression')}
+        _fields = []
+        expressions = {}
+        counter = 1
+        for field in fields:
+            if hasattr(field, 'resolve_expression'):
+                field_id_prefix = getattr(field, 'default_alias', field.__class__.__name__.lower())
+                while True:
+                    field_id = field_id_prefix + str(counter)
+                    counter += 1
+                    if field_id not in field_names:
+                        break
+                expressions[field_id] = field
+                _fields.append(field_id)
+            else:
+                _fields.append(field)
+
+        clone = self._values(*_fields, prepare=True, selects_all=selects_all, **expressions)
```

Comment:
Fyi: this is based on https://github.com/django/django/blob/2fd755b361d3da2cd0440fc9839feb2bb69b027b/django/db/models/query.py#L1318-L1337